examples/HydroValleys/case3.jl

```diff
@@ -32,35 +32,35 @@ data[1]
 params = set_param( stages = 12, 
                     model_constructor_grid  = DCPPowerModel,
                     post_method             = PowerModels.post_opf,
-                    solver                  = ClpSolver());
+                    optimizer               = Clp.Optimizer);
 params
 
 #' ## Build Model
 m = hydrothermaloperation(data, params);
 
 #' ## Solve
-status = solve(m, iteration_limit = 60);
-status
+status = SDDP.train(m.policygraph;iteration_limit = 100);
 
 #' ## Simulation
-srand(1111)
-results = simulate_model(m, 100);
+import Random
+Random.seed!(1111)
+results = HydroPowerModels.simulate(m, 100);
 results
 
 #' ## Testing Results
 #' Objective
-using Base.Test
-@test isapprox(results["simulations"][1]["objective"], 12400.00, atol=1e-2)
+using Test
+@test isapprox(results[:simulations][1][1][:objective], 10496.09, atol=1)
```

Comment:
Hmm. To get the old behaviour you need something like
```julia
sum(s[:stage_objective] for s in results[:simulations][1])
```
We could make this nicer.